### PR TITLE
Update theme version number

### PR DESCRIPTION
**Pull request by katrialesser**
# READY to review but not merge - waiting for 1st pr and new release to be cut

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- 1/20 or today if possible please, very short pr

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by yvonnetangsu — APPROVED** [review]
Approved and will wait and not merge yet :)
